web/src/app/pages/register/internReg.controller.js

```diff
@@ -81,21 +83,56 @@
 
             var attributeList = [];
 
-            var attributes = [{Name: 'email', Value: email}, {Name: 'profile', Value:  '/'}, {Name:'name', Value: 'ADMIN' }];
-            _.each(attributes,function(attribute){
-              console.log(attribute);
-              attributeList.push(new AWSCognito.CognitoIdentityServiceProvider.CognitoUserAttribute(attribute));
-            } );
+            var attributes = [{
+                Name: 'email',
+                Value: email
+            }, {
+                Name: 'profile',
+                Value: '/'
+            }, {
+                Name: 'name',
+                Value: 'ADMIN'
+            }];
+            _.each(attributes, function(attribute) {
+                console.log(attribute);
+                attributeList.push(new AWSCognito.CognitoIdentityServiceProvider.CognitoUserAttribute(attribute));
+            });
 
             console.log(attributeList);
 
             userPool.signUp(username, password, attributeList, null, function(err, result) {
                 if (err) {
-                    //console.log(err);
+
                     return;
                 }
             });
         };
+        $scope.formdata = {};
+
+        $scope.submitted = 'true'; //for form validation
+        $scope.reset = function() {
+            $scope.submitted = false;
+
+            $scope.formdata = {};
+        }
+
+        $scope.University = ["UCSC", "UOM-CSE", "UOM-IT", "University of Kelaniya", "Uva Wellassa University", "University of Rajarata", "University of Peradeniya", "University of Jaffna", "SLIIT", "IIT", "APIIT", "Sabaragamuwa University", "Sri Jayawardanapura University", "Northshore College of Business and Technology", "University of Wayamba", "Auston University", "General Sir John Kotelawala Defence University", "NSBM", "Umea University–Sweden"];
```

Comment:
Anura aiya told to store the data in the front end at the moment. Shall I change?